csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1706,6 +1706,23 @@ Examples:
     cisco-sa-20190513-secureboot
 ```
 
+This value is also used to define the filename for the CSAF document. The following rules MUST be applied to determine the filename for the CSAF document:
```

Comment:
We specify the filename of our CSAF document in a quite hidden section under `Document Property - Tracking - ID`. I would not have expected that as a reader. Can we find some other, more adequate place for this specification so that it can also be found in the TOC?

Reply:
We should track that as a separate issue. Therefore: See #277.